shuup/front/templates/shuup/front/macros/general.jinja

```diff
@@ -257,6 +257,12 @@
             </div>
             <div class="collapse navbar-collapse" id="admin-tools-menu">
                 <ul class="nav navbar-nav navbar-right">
+                {% if object %}
+                    {% set page = shuup.general.get_admin_url(object) %}
+                    <li> 
+                        <a href="{{ page.url }}" target="_blank">{{ _("Edit {page_name} in admin".format(page_name=page.name|truncate(75, True))) }}</a>
```

Comment:
The format here is incorrect. `{{ _("Edit {page_name} in admin").format(page_name=page.name|truncate(75, True)) }}` would be better.